src/resources/help/en_US/relnotes3.3.0.html

```diff
@@ -41,6 +43,7 @@
   <li>A new (beta) <a href="https://en.wikipedia.org/wiki/Names_given_to_the_Spanish_language">Castilian Spanish</a> translation was created thanks to <a href="https://github.com/cessenat">@cessenat</a> using auto-translation from English. It likely still requires review by someone who is fluent in Spanish and we would welcome any volunteers.</li>
   <li>References to old email lists (@ornl.gov) throughout code and documentation were replaced with references to the help pages of the new website.</li>
   <li>The <code>Pick</code> operator now has the option to include the Euclidean distance between the current and previous <code>Pick</code> in the GUI's output window. To enable this option, check the <code>Distance to previous</code> box. You should then see a new field in the tabular output display titled <code>Distance to previous</code>.</li>
+  <li>Support for <i>SeedMe</i> was removed since <i>SeedMe</i> was replaced with <i>SeedMeLab</i> (https://seedmelab.org).</li>
```

Comment:
I wonder if this release note might be a bit confusing...I mean it says we've removed seedme support. But, if seedme was replaced with SeedMeLab, why didn't we just replace our seedme support with seedmelab support?